packages/server/package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prebuild": "rimraf dist",
     "build": "nest build --webpack --webpackPath webpack.config.js",
-    "dev": "cross-env TYPEORM_LOGGING=true nest build --webpack --webpackPath webpack-hmr.config.js",
+    "dev": "cross-env NODE_OPTIONS=--openssl-legacy-provider cross-env TYPEORM_LOGGING=true nest build --webpack --webpackPath webpack-hmr.config.js",
```

Comment:
Is it a typo? we are calling cross-env twice?

Reply:
ah you're right, didn't realise we can combine the two